pkgs/development/python-modules/functiontrace/default.nix

```
@@ -0,0 +1,27 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, toml
```

```suggestion
, setuptools
, toml
```

Bidning required for above review comment